models/fates/inst/template.job.createclone

```diff
@@ -48,47 +51,46 @@ export GFORTRAN_UNBUFFERED_PRECONNECTED=yes
-./xmlchange -file env_mach_pes.xml -id MAX_TASKS_PER_NODE -val 1
-./xmlchange -file env_mach_pes.xml -id TOTALPES -val 1
+#./xmlchange -file env_mach_pes.xml -id NTASKS_ATM -val 1
+#./xmlchange -file env_mach_pes.xml -id NTASKS_LND -val 1
+#./xmlchange -file env_mach_pes.xml -id NTASKS_ICE -val 1
+#./xmlchange -file env_mach_pes.xml -id NTASKS_OCN -val 1
+#./xmlchange -file env_mach_pes.xml -id NTASKS_CPL -val 1
+#./xmlchange -file env_mach_pes.xml -id NTASKS_GLC -val 1
+#./xmlchange -file env_mach_pes.xml -id NTASKS_ROF -val 1
+#./xmlchange -file env_mach_pes.xml -id NTASKS_WAV -val 1
+#./xmlchange -file env_mach_pes.xml -id MAX_TASKS_PER_NODE -val 1
+#./xmlchange -file env_mach_pes.xml -id TOTALPES -val 1
   # Modifying : env_build.xml
-./xmlchange -file env_build.xml -id GMAKE -val make
-./xmlchange -file env_build.xml -id DEBUG -val FALSE
+./xmlchange --file env_build.xml --id GMAKE --val make
+./xmlchange --file env_build.xml --id DEBUG --val FALSE
   # Modifying : env_run.xml
-./xmlchange -file env_run.xml -id REST_N -val 1
-./xmlchange -file env_run.xml -id REST_OPTION -val nyears
-./xmlchange -file env_run.xml -id DATM_CLMNCEP_YR_START -val 1974
-./xmlchange -file env_run.xml -id DATM_CLMNCEP_YR_END -val 2004
-./xmlchange -file env_run.xml -id DIN_LOC_ROOT -val @INDIR@
-./xmlchange -file env_run.xml -id DIN_LOC_ROOT_CLMFORC -val '@INDIR@'
-./xmlchange -file env_run.xml -id DOUT_S -val TRUE
-./xmlchange -file env_run.xml -id DOUT_S_ROOT -val '@CASEDIR@/run'
-./xmlchange -file env_run.xml -id PIO_DEBUG_LEVEL -val 0
-./xmlchange -file env_run.xml -id ATM_DOMAIN_FILE -val 'domain.lnd.@SITE_NAME@.nc'
-./xmlchange -file env_run.xml -id ATM_DOMAIN_PATH -val '@INDIR@/share/domains/domain.clm/'
-./xmlchange -file env_run.xml -id LND_DOMAIN_FILE -val 'domain.lnd.@SITE_NAME@.nc'
-./xmlchange -file env_run.xml -id LND_DOMAIN_PATH -val '@INDIR@/share/domains/domain.clm/'
-./xmlchange -file env_run.xml -id CLM_USRDAT_NAME -val '@SITE_NAME@'
+./xmlchange --file env_run.xml --id REST_N --val 1
+./xmlchange --file env_run.xml --id REST_OPTION --val nyears
+./xmlchange --file env_run.xml --id DATM_CLMNCEP_YR_START --val 1974
+./xmlchange --file env_run.xml --id DATM_CLMNCEP_YR_END --val 2004
+./xmlchange --file env_run.xml --id DIN_LOC_ROOT --val @INDIR@
+./xmlchange --file env_run.xml --id DIN_LOC_ROOT_CLMFORC --val '@INDIR@'
+./xmlchange --file env_run.xml --id DOUT_S --val TRUE
+./xmlchange --file env_run.xml --id DOUT_S_ROOT --val '@CASEDIR@/run'
+./xmlchange --file env_run.xml --id PIO_DEBUG_LEVEL --val 0
+./xmlchange --file env_run.xml --id ATM_DOMAIN_FILE --val 'domain.lnd.@SITE_NAME@.nc'
+./xmlchange --file env_run.xml --id ATM_DOMAIN_PATH --val '@INDIR@/share/domains/domain.clm/'
+./xmlchange --file env_run.xml --id LND_DOMAIN_FILE --val 'domain.lnd.@SITE_NAME@.nc'
+./xmlchange --file env_run.xml --id LND_DOMAIN_PATH --val '@INDIR@/share/domains/domain.clm/'
+./xmlchange --file env_run.xml --id CLM_USRDAT_NAME --val '@SITE_NAME@'
```

Comment:
A lot of this seems redundant across multiple files. Not necessary for this PR, but it would be nice if there was a way to use `source` or something else to keep these commands in only one place.

Reply:
@ashiklom I dont understand this comment, could you elaborate?  This is the approach used to set CLM settings...

Reply:
Also create clone is still a work in progress since I still cannot get the actual CIME command ./create_clone to work and not much help from the CLM community....so its not really implemented yet

Reply:
I just meant that, for example, lines 77-81 of this file are, character for character, identical to lines 74-78 the `template.job` file, so any changes we might want to make in the future would have to be changed twice. Again, not having the perspective on the full code, I don't know how simple it would be to remove this redundancy with some kind of wrapper or template (which is why it's just a comment and not a "Request changes"), but something to think about.

Reply:
@ashiklom Ah, ok.  Well the issue is that we dont want to template.job files but until I can figure out how to properly use ./create_clone or the bugs are sorted we need to continue to rebuild the model exe. But the goal is to deprecate the current template.job for the version that has the create clone command so that we dont rebuild every time but instead re-use the existing exe.  So in the end there are redundancies not but I hope to get rid of one at some point